release.yml

```diff
@@ -0,0 +1,69 @@
+pool:
+  vmImage: 'windows-latest'
+
+variables:
+  solution: '**/*.sln'
+  testProjects: "**/*Tests/*.csproj"
+  publishProject: "**/*Tool/*.csproj"
+  buildConfiguration: 'Release'
+  version: 'Preview'
```

Comment:
Is this version used here?

Reply:
It is only a place holder, use this variable for store the version from task.